packages/suite/src/components/suite/NavigationBar/components/NavigationActions/components/ActionItem/index.tsx

```diff
@@ -61,6 +61,12 @@ const AlertDot = styled.div`
     background: ${colors.NEUE_TYPE_RED};
 `;
 
+// The Icon can be switched dynamically (e.g. Discreet mode icon) and without fixed size the whole row of NavigationActions jumps on the first switch
+const StyledIcon = styled(Icon)`
```

Comment:
shouldn't this be fixed in `Icon` component? 
there might be other places where this jumping happens.
i see that `Icon` component `SvgWrapper` already sets height based on Icon.size prop, should be enough to put the width as well, no?
@slowbackspace ?

Reply:
I was thinking about it, but I don't want to introduce new bugs right now :/

Reply:
yeah, thats why i've tagged @slowbackspace to confirm that.
correct me if im wrong but i think all icons are square shaped

Reply:
I can create a new PR with global change for the next release right after this is merged.

Reply:
ok, i've created an issue for that https://github.com/trezor/trezor-suite/issues/2868